readme.md

```diff
@@ -83,7 +83,11 @@ require('@zeit/ncc')('/path/to/input', {
   watch: false, // default
   v8cache: false, // default
   quiet: false, // default
-  debugLog = false // default
+  debugLog = false, // default
+  // https://webpack.js.org/configuration/resolve/#resolvealias
+  resolveAlias: {}, // default
+  // https://webpack.js.org/configuration/resolve/#resolvemodules
+  resolveModules: [] // default
```

Comment:
I think we want to avoid leaking webpack details through to the ncc API in case we need to swap webpack for rollup or some other alternative.

Reply:
Other tools likely have similar options; Rollup for sure does.

Changing build tool would definitely be a breaking change, if it ever happens, so I don't particularly see the issue with supporting a particular flavor of options.

This option would reduce my bundle by dropping 2 unused heavy sub-dependencies. I just realized that my built file is 1.2MB due a couple of dependencies I don't really use.